exporter/otlp-metrics/test/opentelemetry/exporter/otlp/metrics_exporter_test.rb

```diff
@@ -97,6 +106,8 @@
     it 'prefers explicit parameters rather than the environment' do
       exp = OpenTelemetry::TestHelpers.with_env('OTEL_EXPORTER_OTLP_ENDPOINT' => 'https://localhost:1234',
                                                 'OTEL_EXPORTER_OTLP_CERTIFICATE' => '/foo/bar',
+                                                'OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE' => METRICS_CLIENT_CERT_A_PATH,
+                                                'OTEL_EXPORTER_OTLP_CLIENT_KEY' => METRICS_CLIENT_CERT_A_PATH,
```

Comment:
For this test, I think we need a second certificate to verify `client_certificate_file` and `client_key_file` passed as arguments supersede the environment variables. 

https://github.com/open-telemetry/opentelemetry-ruby/blob/d9324e03f8520c35baf1fcaeb0b96d50ba4e923c/exporter/otlp/test/opentelemetry/exporter/otlp/exporter_test.rb#L110-L140

Reply:
Thanks, I have added that